example/pubspec.yaml

```diff
@@ -21,9 +21,10 @@ dev_dependencies:
     path: ../
 
   pedantic: ^1.8.0
-  e2e: ^0.2.1
```

Comment:
This has been deprecated and replaced with [integration_test](https://github.com/flutter/plugins/tree/master/packages/integration_test), but I actually don't think we need it.